docker-compose.deps.yml

```diff
@@ -0,0 +1,22 @@
+version: '2'
+
+services:
+   pip:
+     extends:
+       file: services.yml
+       service: base
+     command: pip install -r requirements.txt --pre
+     volumes_from:
+       - static
+   assets:
+     extends:
+       file: services.yml
+       service: base
+     command: scripts/clean_assets
+     volumes_from:
+       - static
+  static:
```

Comment:
Here I get an error about indentation
```shell
ERROR: yaml.parser.ParserError: while parsing a block mapping
  in "./docker-compose.deps.yml", line 1, column 1
expected <block end>, but found '<block mapping start>'
  in "./docker-compose.deps.yml", line 18, column 3
```
which is easy to fix, but I wonder why your docker wasn't complaining.
```shell
$ docker --version
Docker version 1.10.3, build 20f81dd
```